esti/ugc_test.go

```diff
@@ -301,7 +300,8 @@ func listUnderlyingStorage(t *testing.T, ctx context.Context, s3Client *s3.S3, b
 	for _, obj := range listOutput.Contents {
 		objects = append(objects, Object{
 			Key:          fmt.Sprintf("s3://%s/%s", bucket, aws.StringValue(obj.Key)),
-			LastModified: *obj.LastModified})
+			LastModified: *obj.LastModified,
+		})
```

Comment:
Just to be on the safe side
```suggestion
			LastModified: aws.TimeValue(obj.LastModified),
		})		
```

Reply:
without the space padding